coq/coq-syntax.el

```diff
@@ -1070,7 +1070,7 @@ It is used:
 ;; It is understood here as being a goal.  This is important for
 ;; recognizing global identifiers, see coq-global-p.
 (defconst coq-save-command-regexp-strict
-  (concat "\\`\\(?:Time\\s-+\\)?\\("
+  (concat "\\`\\(?:Time\\s-+\\|Timeout\\s-+\\(?:[[:digit:]]+\\s-+\\)?\\)?\\("
```

Comment:
```suggestion
  (concat "\\`\\(?:Time\\s-+\\|Timeout\\s-+[[:digit:]]+\\s-+\\)?\\("
```
The number is mandatory after `Timeout`.  (What's up with `\\s-+`, though?  what's the `-`?)

Reply:
> The number is mandatory after `Timeout`

good catch, thanks

>  what's the `-`?

this is specific to GNU Emacs regexps:

https://www.gnu.org/software/emacs/manual/html_node/emacs/Regexp-Backslash.html#Regexp-Backslash

> **`\sc`**  
>     matches any character whose syntax is **`c`**. Here **`c`** is a character that designates a particular syntax class: thus, ‘w’ for word constituent, ‘-’ or ‘ ’ for whitespace, ‘.’ for ordinary punctuation, etc.